cpp/ps0001/p0056.so1.cpp

```diff
@@ -4,13 +4,14 @@ using namespace std;
 
 int main()
 {
-
-    int x , y , o ;
-
+    // prompting user for 3 numbers to add 1/2
+    unsigned short num1, num2, num3;
     cout << "please enter x and y and o \n";
-    cin >> x >> y >> o ;
+    cin >> num1 >> num2 >> num3;
 
-    cout << "total = " << x + y + o << "\n" ;
+    // outputting sum of the 3 numbers 3/3
+    cout << "total = " << num1 + num2 + num3 << "\n" ;
 
     return 0;
 }
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code